Content.Shared/FeedbackSystem/FeedbackPopupPrototype.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Robust.Shared.Prototypes;
+
+namespace Content.Shared.FeedbackSystem;
+
+/// <summary>
+/// Prototype that describes the contents of a feedback popup.
+/// </summary>
+[Prototype]
+public sealed partial class  FeedbackPopupPrototype : IPrototype
```

Comment:
THE HORRRORRRRR

```suggestion
public sealed partial class FeedbackPopupPrototype : IPrototype
```
(jk <3)